api/src/functions/express/index.ts

```diff
@@ -41,6 +41,13 @@ export default (cognitoArn: string, vpcConfig: FnType["vpc"]): FnType => {
           cors: false,
         },
       },
+      {
+        http: {
+          method: "POST",
+          path: "/api/users/emailCheck",
+          cors: true,
```

Comment:
I'm pretty sure this is required and `api.account.business.nj.gov` with the different subdomain requires CORS, but just wanted to double check w/ folks.